assets/app/scripts/controllers/util/error.js

```diff
@@ -31,4 +31,8 @@ angular.module('openshiftConsole')
     if (params.error_description) {
       $scope.errorDetails = params.error_description;
     }
+    
+    $scope.reloadConsole = function() {
+      $window.location.href = "/";
```

Comment:
Tried it in the built binary where there is a context and it seemed like it
was working correctly.
On Apr 18, 2016 5:08 PM, "Jordan Liggitt" <notifications@github.com> wrote:

> In assets/app/scripts/controllers/util/error.js
> <https://github.com/jwforres/origin/pull/38#discussion_r60134535>:
>
> > @@ -31,4 +31,8 @@ angular.module('openshiftConsole')
> >      if (params.error_description) {
> >        $scope.errorDetails = params.error_description;
> >      }
> > +
> > +    $scope.reloadConsole = function() {
> > +      $window.location.href = "/";
>
> does this work with subcontexts?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jwforres/origin/pull/38/files/af570740dc6ecf32ce69439fbd08b88cc600794f#r60134535>
>